### DISCUSSION TITLE
Ajuda com a Instalasão..

### DISCUSSION THREAD

**Discussion by JetLi-Almeida** [RamonSilva20/mapos#2168] [General]
Tentando fazer a instalação aqui e não consigo passa do Composer..
estou usando o XAMPP em sua ultima versão, tentei versões antigas mas mesmo problema..
Composer tambem esta instalado o do tutorial mas ainda sim da erro.

![image](https://github.com/RamonSilva20/mapos/assets/79049787/8a78326b-d866-4795-acce-932a4bfe278e)

**Comment by LucasDapper** [comment]
Bom dia!

Assiste o vídeo do Edson, segue o passo a passo que resolvera esse problema...


Segue link: https://www.youtube.com/watch?v=mvkYfwLhWUM&t=34s

**Reply by JetLi-Almeida** [reply]
Obrigado, consegui resolver o problema.. mas agora tô com outro na hora de imprimir a O.S da erro. Vou adicionar um print.

![image](https://github.com/RamonSilva20/mapos/assets/79049787/f98366f7-cb7e-41c3-9b09-9a7b5d66ea0c)
